tests/test_draw.py

```diff
@@ -0,0 +1,56 @@
+# tests.test_draw
+# Tests for the high-level drawing utility functions
+#
+# Author:  Benjamin Bengfort <benjamin@bengfort.com>
+# Created: Sun Aug 19 11:21:04 2018 -0400
+#
+# ID: test_draw.py [] benjamin@bengfort.com $
+
+"""
+Tests for the high-level drawing utility functions
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from yellowbrick.draw import *
+from .base import VisualTestCase
+
+
+##########################################################################
+## Visual Test Cases for Drawing Utilities
+##########################################################################
+
+class TestDraw(VisualTestCase):
+    """
+    Visual tests for the high-level drawing utilities
+    """
+
+    def test_manual_legend(self):
+        """
+        Check that the manual legend is drawn without axes artists
+        """
+        # Draw a random scatter plot
+        random = np.random.RandomState(42)
+
+        Ax, Ay = random.normal(50, 2, 100), random.normal(50, 3, 100)
+        Bx, By = random.normal(42, 3, 100), random.normal(44, 1, 100)
+        Cx, Cy = random.normal(20, 10, 100), random.normal(30, 1, 100)
+
+
+        _, ax = plt.subplots()
+        ax.scatter(Ax, Ay, c='r', alpha=0.35, label='a')
+        ax.scatter(Bx, By, c='g', alpha=0.35, label='b')
+        ax.scatter(Cx, Cy, c='b', alpha=0.35, label='c')
+
+        # Add the manual legend
+        manual_legend(
+            ax, ('a', 'b', 'c'), ('r', 'g', 'b'), frameon=True, loc='upper left'
+        )
```

Comment:
Great test and good to see these matplotlib kwargs work as expected!

Reply:
And on Windows too, how'd that happen!?

Reply:
Well, still trying to configure the Surface loaner...